clients/fides-js/src/lib/consent-value.ts

```diff
@@ -23,16 +23,16 @@ export const resolveLegacyConsentValue = (
 
 export const resolveConsentValue = (
   notice: PrivacyNotice,
-  context: ConsentContext
+  context: ConsentContext,
+  previouslyConsented?: boolean
 ): boolean => {
-  if (notice.current_preference) {
-    return transformUserPreferenceToBoolean(notice.current_preference);
-  }
   if (notice.consent_mechanism === ConsentMechanism.NOTICE_ONLY) {
     return true;
   }
   const gpcEnabled =
-    !!notice.has_gpc_flag && context.globalPrivacyControl === true;
+    !!notice.has_gpc_flag &&
+    context.globalPrivacyControl === true &&
+    !previouslyConsented;
```

Comment:
why would previously consenting affect `gpcEnabled`?

Reply:
`gpcEnabled` is probably not the best var name here as it's diff than `gpcEnabled` on the notice itself. Here it's more like "eligible to be overridden by gpc". If user has previously given consent for this notice, then we don't want to override their previous selection. 

Here's an example:

1. User navs to the site with GPC enabled in browser
2. Fides has a GPC-enabled notice, surfaces the banner with that notice disabled by default and has the label `applied` on the modal. 
3. User decides to consent to this notice, so that the label now says `overridden` (GPC is enabled but consent has been set to override the configured default)
4. Next time the user navs to the site, we ensure we respect the user's choice